include/seqan3/range/concept.hpp

```diff
@@ -135,4 +135,23 @@ SEQAN3_CONCEPT pseudo_random_access_range =
     pseudo_random_access_iterator<std::ranges::iterator_t<rng_t>>;
 //!\endcond
 
+/*!\interface seqan3::sequence <>
+ * \brief The generic concept for a sequence.
+ * \ingroup range
+ * \extends std::ranges::input_range
+ *
+ * We define a range over an alphabet as a *sequence*.
+ * Thus, a type models the concept seqan3::sequence if it is at least an std::ranges::input_range
```

Comment:
```suggestion
 * A type models seqan3::sequence if it is at least an std::ranges::input_range
```
And then I would remove the sentence above because they are literally the same.